content/intro/pricing/rates.md

```diff
@@ -0,0 +1,94 @@
+---
+menu:
+  main:
+    parent: pricing
+title: Rates
+weight: 10
+---
+
+cloud.gov fees are broken into three components:
+
+- Access package
+- Usage
+- Managed services
+
+[Find out more about this pricing model.]({{< relref "intro/pricing/pricing-model.md" >}})
```

Comment:
The pricing model page still has a lot of related info on it that doesn't have a better home, eg that it's a cost-recovery model, that it's still in flux and needs feedback, that we want people to think about the money it's saving them in personnel and compliance delays, etc. I think I want to slim down the pricing model page to *just* focus on those points, and leave the rest of the model definition to the terminology page.